docs/pages/push-notifications/sending-notifications.mdx

```diff
@@ -264,6 +264,7 @@ Each message must be a JSON object with the given fields (only the `to` field is
 | Field            | Platform        | Type                                         | Description                                                                                                                                                                                                                                                                                                                                                       |
 | ---------------- | --------------- | -------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `to`             | Android and iOS | <CODE>string \| string[]</CODE>              | An Expo push token or an array of Expo push tokens specifying the recipient(s) of this message.                                                                                                                                                                                                                                                                   |
+| `_contentAvailable` | iOS Only     | <CODE>boolean \| undefined</CODE>            | When this is set to true, the notification will cause the iOS app to be started in the background to run a [background task](/versions/unversioned/sdk/notifications/#background-notifications).                                                                                                                                                                  |                                           
```

Comment:
```suggestion
| `_contentAvailable` | iOS Only     | <CODE>boolean \| undefined</CODE>            | When this is set to true, the notification will cause the iOS app to start in the background to run a [background task](/versions/latest/sdk/notifications/#background-notifications).                                                                                                                                                                  |                                           
```

The link should direct to the latest version of Notifications API reference than the unversioned/Next doc.

Reply:
These changes are already in SDK 50 and 51, this is actually correcting the docs.